kolibri_explore_plugin/assets/src/views/ContentPage.vue

```diff
@@ -41,6 +41,9 @@
           <p v-if="content.author">
             {{ $tr('author', { author: content.author }) }}
           </p>
+          <p v-if="content.tags">
```

Comment:
if `content.tags` is an empty list this will be shown anyway because `Boolean([]) === true`,  but in any case, I think it's okay for this initial version.

Reply:
True, maybe we should improve the comparison.